src/corehost/common/pal.windows.cpp

```diff
@@ -130,8 +130,26 @@ bool pal::get_default_packages_directory(string_t* recv)
     {
         return false;
     }
-    append_path(&*recv, _X(".nuget"));
-    append_path(&*recv, _X("packages"));
+    append_path(recv, _X(".nuget"));
+    append_path(recv, _X("packages"));
+    return true;
+}
+
+bool pal::get_default_extensions_directory(string_t* recv)
+{
+    recv->clear();
+
+#ifdef _TARGET_X86_
```

Comment:
I think this check should instead use FOLDERID_ProgramFilesX86 value as described at https://msdn.microsoft.com/en-us/library/windows/desktop/dd378457(v=vs.85).aspx. It will return PF-x86 for 32bit app on 64bit and PF for 32bit app on 32bit OS.

We shouldnt rely on environment variables for this.